substrate/runtime-io/with_std.rs

```diff
@@ -102,7 +102,7 @@ pub fn storage_root() -> H256 {
 
 /// A trie root formed from the enumerated items.
 pub fn enumerated_trie_root(serialised_values: &[&[u8]]) -> [u8; 32] {
```

Comment:
this should be prefixed with `keccak_` to make its function specific. (i.e. renamed to `keccak_enumerated_trie_root`).

Reply:
I thought the plan was to switch to using Blake2b soon – next 30 - 60 days? – and unless we have multiple crypto/format combos, is it really useful to have them explicitly named?